src/components/FormElements/FormFieldInput/FormFieldInput.js

```diff
@@ -81,6 +82,7 @@ const FormFieldInput = ({
         name={id}
         value={value}
         showSingleHiddenInputValue
+        renderTime={renderTime}
```

Comment:
PR is great - just need to add a test for this prop 👍

Reply:
done